spec/Section 3 -- Type System.md

```diff
@@ -2110,9 +2113,13 @@ behavior of [custom scalar types](#sec-Scalars.Custom-Scalars). The URL should
 point to a human-readable specification of the data format, serialization, and
 coercion rules. It must not appear on built-in scalar types.
 
+Note: Details on implementing a GraphQL scalar specification can be found in the [scalars.graphql.org implementation guide](https://scalars.graphql.org/implementation-guide).
+
 In this example, a custom scalar type for `UUID` is defined with a URL pointing
 to the relevant IETF specification.
 
 ```graphql example
 scalar UUID @specifiedBy(url: "https://tools.ietf.org/html/rfc4122")
+scalar URL @specifiedBy(url: "https://tools.ietf.org/html/rfc3986")
+scalar DateTime @specifiedBy(url: "https://scalars.graphql.org/andimarek/date-time")
```

Comment:
The text before this block says "In this example, a custom scalar type for UUID is defined with a URL pointing to the relevant IETF specification." therefore the block should only define UUID. The prior examples are sufficient, no need for change here.

```suggestion
```